code/API_definitions/CAMARA Connectivity Insights API.yaml

```diff
@@ -171,230 +282,129 @@
             "200":
               description: policy has been deleted
 
-      /network-insight/device:
+      /subscriptions:
```

Comment:
replacing subscriptions related API to use webhooks for notifications

Reply:
My preference is for a request for network insights (live snapshot of network confidence) to be distinct to a subscription (ongoing notifications of network confidence for the session) - we can discuss on the call.

Reply:
discussed during the call and non subscription based insights is available as part of a dedicated post call.